### ISSUE TITLE
Can not build framework_objc because of error "This frameworks value is wrong"

### ISSUE THREAD

**Issue by kodakvn** [HackWebRTC/webrtc#31]
I am trying to build webrtc for iOS to use recording module but faced with error:

ERROR at //sdk/BUILD.gn:452:9: This frameworks value is wrong.All listed frameworks names must not include any
path component and have ".framework" extension.
        libs += [
        ^--------------
See //sdk/BUILD.gn:435:5: whence it was called.
    rtc_library("video_objc") {
    ^--------------------------
See //BUILD.gn:468:7: which caused the file to be included.
      "sdk",
      ^----

this because of these lines in //sdk/BUILD.gn
if (rtc_use_recorder) {
    libs += [
    "z",
    "bz2",
    "iconv",
    "//sdk/libs/ffmpeg/lib/iOS/libswresample.a",
    "//sdk/libs/ffmpeg/lib/iOS/libavformat.a",
    "//sdk/libs/ffmpeg/lib/iOS/libavcodec.a",
    "//sdk/libs/ffmpeg/lib/iOS/libavutil.a",
    ]
}

Anyone knows how to fix this error?
Thank you very much!

**Comment by kodakvn** [comment]
How can I build HackWebRTC with target framework_objc for iOS?
I have read README but nothing.
It's seems that ".gclient" & ".gclient_entries" files are missing when comparing with "webrtc_ios" cloned from google by command "fetch --nohooks webrtc_ios".

**Comment by Piasy** [comment]
You need follow google's guide to setup build env and project. Then you can add this repo as a remote, and switch to it, then sync & build it.

**Comment by kodakvn** [comment]
> You need follow google's guide to setup build env and project. Then you can add this repo as a remote, and switch to it, then sync & build it.

@Piasy 
But which commit from google should I checkout/base to sync up with HackWebRTC?
By sync up with latest commit, there are too many conflicts.
Thank you very much!

**Comment by Piasy** [comment]
You can check the commit history of this repo to find it out.

**Comment by kodakvn** [comment]
I have tried to sync up HackWebRTC I got error message like this

ERROR at //sdk/BUILD.gn:159:7: Unsupported value in libs.
      "AVFoundation.framework",
      ^-----------------------
Use frameworks to list framework dependencies.
See //sdk/BUILD.gn:141:3: whence it was called.
  rtc_library("helpers_objc") {
  ^----------------------------
See //BUILD.gn:475:7: which caused the file to be included.
      "sdk",

In case of changing "libs" to "frameworks" (I think this is new definition), above error appears again: 
"This frameworks value is wrong.All listed frameworks names must not include any path component and have ".framework" extension."
